src/app/components/Brand/README.md

```diff
@@ -0,0 +1,12 @@
+# Header [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Brand)	
+
+## Description	
+The `Brand` component provides the BBC logo (as SVG), nested inside a styled span, link and div. The link is currently hardcoded to "https://www.bbc.co.uk/news". `Brand` takes a `brandName` as a prop. This prop is passed to a [VisuallyHiddenText](./../VisuallyHiddenText) component, nested inside Brand.
```

Comment:
```suggestion
The `Brand` component provides the BBC News logo (as SVG), nested inside a styled span, link and div. The link is currently hardcoded to "https://www.bbc.co.uk/news". `Brand` takes a `brandName` as a prop. This prop is passed to a [VisuallyHiddenText](../VisuallyHiddenText) component, nested inside Brand. Note that this does not currently affect the branding itself, which always renders as `BBC NEWS`.
```

We should raise an issue to render different brand SVG and URL based on `brandName`. 🤔

Reply:
I think this relates to Simorgh's roadmap w/r/t different languages, worth brining this up regarding milestones at our next planning session. @sareh @jimjohnsonrollings